NOM_visualizations/R/NOM_R_notebook.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>So I think input from Yuri was that we should keep confidence scores &gt; 0.3, correct?  </p><p> </p><p>For determining which samples to keep I would take a different approach. Instead of looking for files with a certain % assigned, I'd look for files with a certain # assigned. If you have a bad sample with only 5 peaks (but all five are assigned bc they're contaminants, say), your filter would keep this file, and mine would dump it.  &lt;- I gave this same feedback to @<a href="https://github.com/samobermiller" target="_blank">samobermiller</a>&nbsp;and maybe I'm wrong, but I think it's worth revisiting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
I think this is worth talking about more, how do we know what a good minimum number of peaks is?